lua/orgmode/utils.lua

```diff
@@ -261,7 +261,7 @@ function utils.get_ts_matches(query, node, file_content)
 end
 
 function utils.get_node_text(node, content)
-  if not node then
+  if not node or node == '' then
```

Comment:
When is node empty string? This function should either get a ts node or nil.

Reply:
I don't really know why this happened. This is unrelated to the rest of my changes.
It had a state where this throws errors, even on the current `tree-sitter` branch.

I'll see if I can reproduce this, and see where it comes from

Reply:
Yeah, please try to reproduce it. If it's not related to this PR, i'll fix it separately.

Reply:
It happens when there is an empty headline.

just opening a file like this throws.

```org
*
```

Reply:
Thanks, I'll take a look.

Reply:
Fixed it. I was using default as empty string instead of nil for todo keyword node, which is missing in this case.